src/lib.rs

```diff
@@ -1,49 +1,41 @@
-//! Laminar semi-reliable UDP protocol for multiplayer games. This library implements wraps around a UDP
-//! and provides light weight stream based interface that provides certain guarantees like reliability.
+//! Laminar is a semi-reliable UDP-based protocol for multiplayer games. This library implements wrappers around the UDP-protocol,
```

Comment:
we should align this with the readme changes, which it seems you have but just leaving this to remind you to make one last check here.